crates/uv-python/python/get_interpreter_info.py

```diff
@@ -463,21 +463,19 @@ def get_operating_system_and_architecture():
 
         musl_version = _get_musl_version(sys.executable)
         glibc_version = _get_glibc_version()
+
         if musl_version:
             operating_system = {
                 "name": "musllinux",
                 "major": musl_version[0],
                 "minor": musl_version[1],
             }
-        elif glibc_version != (-1, -1):
+        else:
             operating_system = {
                 "name": "manylinux",
                 "major": glibc_version[0],
                 "minor": glibc_version[1],
             }
-        else:
-            print(json.dumps({"result": "error", "kind": "libc_not_found"}))
-            sys.exit(0)
```

Comment:
```python
        elif glibc_version != (-1, -1):
            operating_system = {
                "name": "manylinux",
                "major": glibc_version[0],
                "minor": glibc_version[1],
            }
        elif hasattr(sys, "getandroidapilevel"):
            operating_system = {
                "name": "android",
                "major": sys.getandroidapilevel(),
                "minor": 0,
            }
        else:
            print(json.dumps({"result": "error", "kind": "libc_not_found"}))
            sys.exit(0)
```
...
```python
    elif os_and_arch["os"]["name"] == "android":
        manylinux_compatible = False
```

Rather than allowing passing through the glibc version as (-1, -1), I wonder if it's worthwhile considering using [sys.getandroidapilevel](https://docs.python.org/3/library/sys.html#sys.getandroidapilevel) here since it's compatible since 3.7+.
[PEP-738](https://peps.python.org/pep-0738/) also introduced [platform.android_ver](https://docs.python.org/3/library/platform.html#platform.android_ver), but sadly it's only available in 3.13.